crates/parser/src/grammar/patterns.rs

```diff
@@ -386,3 +389,16 @@ fn box_pat(p: &mut Parser) -> CompletedMarker {
     pattern_single(p);
     m.complete(p, BOX_PAT)
 }
+
+// test const_block_pat
+// fn main() {
+//     let const { 15 } = ();
+//     let const { foo(); bar() } = ();
+// }
+fn const_block_pat(p: &mut Parser) -> CompletedMarker {
+    assert!(p.at(T![const]));
+    let m = p.start();
+    p.bump(T![const]);
+    block_expr(p);
```

Comment:
```suggestion
    expressions::block_expr(p);
```

parser generally has only `use super::*`